doc/customization_steps.rst

```diff
@@ -0,0 +1,114 @@
+
+3. Assess `natura.tiff` raster
+--------------------------------
+
+The ``natura.tiff`` raster file delineates areas designated as protected and reserved nature zones. These zones impose land use restrictions, influencing the calculation of renewable potential within a given geographical region.
+The default ``natura.tiff`` is now a CC0 data: Sosa Arango, Chrystian Camilo, 2020, "Protected areas (WDPA)", `<https://doi.org/10.7910/DVN/XIV9BL>`__, Harvard Dataverse, V1.
+By activating the ``natura`` option and incorporating the ``natura.tiff`` file, users can account for protected and reserved nature areas, where renewable assets cannot be installed:
+
+.. code:: yaml
+
+    renewable:
+        onwind:
+            natura: true
+
+.. note::
+
+    By default, the option ``natura: true`` is set for all renewable sources.
+
+It's worth noting that a pre-built ``natura.tiff`` file is included in the dataset and downloaded through ``retrieve_databundle_light`` rule, offering a global coverage of protected areas. This default file serves as a foundational resource for model execution.
+
+While the default file suffices for most analyses, users seeking the latest updates and revisions can opt to enable the ``build_natura_raster`` rule:
+
+.. code:: yaml
+
+    enable:
+        build_natura_raster: true
+
+However, it's worth noting that generating a customized ``natura.tiff`` file with the latest data may require a significant amount of time due to processing complexities.
+
+4. Model Validation
+-------------------
+
+To validate the data obtained with PyPSA-Earth, we recommend to go through the procedure here detailed. An exampled of the validation procedure is available in the `Nigeria validation <https://github.com/pypsa-meets-earth/documentation/blob/main/notebooks/validation/validation_nigeria.ipynb>`_ notebook. Public information on the power system of Nigeria are compared to those obtained from the PyPSA-Earth model.
+
+Simulation procedure
+^^^^^^^^^^^^^^^^^^^^
+
+It may be recommended to check the following quantities in the validation:
+
+#. Inputs used by the model:
+
+    #. Network characteristics
+
+    #. Substations
+
+    #. Installed generation by type
+
+#. Outputs of the simulation:
+
+    #. Demand
+
+    #. Energy mix
+
+Where to look for reference data
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Data availability for many parts of the world is still quite limited. Usually the best sources to compare with are regional data hubs. There is also a collection of harmonized datasets curated by the international organisations. A non-exhaustive list of helpful sources:
+
+* `World Bank <https://energydata.info/>`_;
+
+* International Renewable Energy Agency `IRENA <https://pxweb.irena.org/pxweb/en/IRENASTAT/IRENASTAT__Power%20Capacity%20and%20Generation/ELECCAP_2022_cycle2.px/>`_;

```

Comment:
Hello @yerbol-akhmetov! Great to see that the work is close to be finalised.
As a little additional comment: url of the IRENA databese has changed, as @swaechchha has shared in [her comment](https://github.com/pypsa-meets-earth/pypsa-earth/issues/815#issuecomment-1971624616).

The actual page address is https://pxweb.irena.org/pxweb/en/IRENASTAT/. Could you please replace it?

Reply:
Thanks @swaechchha and @ekatef for noting it. I have updated the url.

Reply:
Perfect, thanks a lot @yerbol-akhmetov!